configure.d/1_global_page_state.conf

```diff
@@ -7,42 +7,33 @@
 . $(dirname $3)/conf_framework
 
 check() {
-	cur_name=$(basename $2)
-	config_file_path=$1
-	if compile_module $cur_name "si_mem_available()" "linux/mm.h"
-	then
-		echo $cur_name "1" >> $config_file_path
-
-	elif compile_module $cur_name "global_zone_page_state(1)" "linux/mm.h"
-	then
-		echo $cur_name "2" >> $config_file_path
-	elif compile_module $cur_name "global_page_state(1)" "linux/mm.h"
-	then
-		echo $cur_name "3" >> $config_file_path
-	else
-	    echo $cur_name "X" >> $config_file_path
-	fi
+    cur_name=$(basename $2)
+    config_file_path=$1
+    if compile_module $cur_name "global_zone_page_state(1)" "linux/mm.h"
+    then
+        echo $cur_name "1" >> $config_file_path
+        elif compile_module $cur_name "global_page_state(1)" "linux/mm.h"
+        then
```

Comment:
To be consistent I must pick this one here as well - too much indent here also. ;)

Reply:
Fixed.